inc/class-wpseo-utils.php

```diff
@@ -1097,4 +1097,31 @@ public static function is_metabox_active( $identifier, $type ) {
 	public static function is_woocommerce_active() {
 		return class_exists( 'Woocommerce' );
 	}
+
+	/**
+	 * Determines whether the plugin is active for the entire network.
+	 *
+	 * @return bool Whether or not the plugin is network-active.
+	 */
+	public static function is_plugin_network_active() {
+		static $network_active = null;
+
+		if ( ! is_multisite() ) {
+			return false;
+		}
+
+		// Store the result in a variable to prevent multiple lookups.
+		if ( null === $network_active ) {
+			$network_active_plugins = wp_get_active_network_plugins();
+
+			if ( in_array( WP_PLUGIN_DIR . '/' . WPSEO_BASENAME, $network_active_plugins, true ) || 0 === strpos( wp_normalize_path( WPSEO_FILE ), wp_normalize_path( WPMU_PLUGIN_DIR ) ) ) {
```

Comment:
Please refrain from using Yoda-style if-statements. Additionally, I'm not sure which of the two would be faster to execute first, but I think executing the `strpos` check first is a bit more efficient.

Reply:
Good point, if there are many network-active plugins, the `strpos` check should indeed be faster.